internal/db/schema/migrations/oss/postgres_28_02_test.go

```diff
@@ -1,151 +0,0 @@
-package oss_test
-
-import (
-	"context"
-	"testing"
-
-	"github.com/hashicorp/boundary/internal/authtoken"
-	"github.com/hashicorp/boundary/internal/db"
-	"github.com/hashicorp/boundary/internal/db/common"
-	"github.com/hashicorp/boundary/internal/db/schema"
-	"github.com/hashicorp/boundary/internal/host/static"
-	"github.com/hashicorp/boundary/internal/iam"
-	"github.com/hashicorp/boundary/internal/kms"
-	"github.com/hashicorp/boundary/internal/session"
-	"github.com/hashicorp/boundary/internal/target"
-	"github.com/hashicorp/boundary/internal/target/tcp"
-	"github.com/hashicorp/boundary/internal/types/resource"
-	"github.com/hashicorp/boundary/testing/dbtest"
-	"github.com/stretchr/testify/require"
-)
-
-func TestMigrations_SessionStateTrigger(t *testing.T) {
```

Comment:
@irenarindos - As you can see, this migration test was deleted since it used the new KMS type, but it doesn't have the required schema changes in the migrations you ran.  If you'd prefer, we can keep "undelete" it and update the migrations to include the kms schemas which is 29002 I believe.